### ISSUE TITLE
Nan errors on a lot of Japanese video

### ISSUE THREAD

**Issue by 120bits** [tomchang25/whisper-auto-transcribe#53]
Hi, first of all nice project! I can transcribe quite a few videos but more often than not they fail with the following exception (I'm using the cli so input is consistent). I couldn't figure out why though. Any suggestions? 

An error occurred during transcription: Expected parameter logits (Tensor of shape (1, 51865)) of distribution Categorical(logits: torch.Size([1, 51865])) to satisfy the constraint IndependentConstraint(Real(), 1), but found invalid values:
tensor([[nan, nan, nan,  ..., nan, nan, nan]], device='cuda:0')
